src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -28,11 +31,50 @@ public Climber() {
 
   public void configure() {
     climberMotor.configFactoryDefault();
+    climberMotor.setInverted(true);
   }
 
   @Override
   public void periodic() {
+    SmartDashboard.putNumber("Climber Encoder", getClimberMotorEncoderCount());
+    SmartDashboard.putBoolean("Climber Down", getClimberBottomSwitch());
     // This method will be called once per scheduler run
 
   }
+
+  public double getClimberMotorEncoderCount() {
+    return -climberMotor.getSelectedSensorPosition();
+  }
+
+  public boolean getClimberBottomSwitch() {
+    return !climberBottomSafetySwitch.get();
+  }
+
+  public void setClimberMotorEncoderCount(double encoderCount) {
+    climberMotor.setSelectedSensorPosition(encoderCount);
+  }
+
+  public void setClimbSpeed(double speed) {
+    double climberSpeed = speed;
```

Comment:
Unnecessary to create a second variable but it will work this way. No changes needed.